pages/download.py

```diff
@@ -285,24 +285,19 @@ def __init__(self):
     def get_content(self):
         # License: GFDL (from old freenetproject.org website)
         return text(md(_("""
-For best performance, Freenet will run continually. It should
-not interfere with your computer usage, as it requires around 
-200MB of RAM and 10%% of one CPU core, plus some disk access. We 
-strongly recommend you shut down Freenet while playing computer 
-games etc. On Windows you can do this from the system tray icon, 
-on other systems use the links on the system menu or the desktop.
+If you know other people who also use Freenet, you can
+[add them as Friends][url_addfriend]. This will make you
+safer against attacks on infrastructure of the Freenet Project
```

Comment:
Consider "Freenet Project infrastructure" instead. Sounds a bit odd to me saying of the Freenet Project.